src/core/src/ffi/minhash.rs

```diff
@@ -58,6 +58,31 @@ unsafe fn kmerminhash_add_sequence(ptr: *mut SourmashKmerMinHash, sequence: *con
 }
 }
 
+ffi_fn! {
+unsafe fn kmerminhash_seq_to_hashes(ptr: *mut SourmashKmerMinHash, sequence: *const c_char, force: bool, size: *mut usize) ->
+Result<*const u64> {
+
+    let mh = SourmashKmerMinHash::as_rust_mut(ptr);
+
+    // FIXME: take buffer and len instead of c_char
+    let c_str = {
+        assert!(!sequence.is_null());
+
+        CStr::from_ptr(sequence)
+    };
```

Comment:
This `FIXME` comment is because I changed a bit how to take buffers/strings later in sourmash, opting for being more explicit about the size (instead of depending on `\0`, like C strings). Check the [`nodegraph_from_buffer`](https://github.com/sourmash-bio/sourmash/blob/2542e6c58b6aa2342df059c56b46ed30442800cc/src/core/src/ffi/nodegraph.rs#L180L185) function, which will be something like this here:

```suggestion
unsafe fn kmerminhash_seq_to_hashes(ptr: *mut SourmashKmerMinHash, sequence: *const c_char, insize: usize, force: bool, size: *mut usize) ->
Result<*const u64> {

    let mh = SourmashKmerMinHash::as_rust_mut(ptr);

    let buf = {
        assert!(!ptr.is_null());
        slice::from_raw_parts(sequence as *const u8, insize)
    };
```

Reply:
Nice! got it, thanks Luiz.